enterprise/actions.py

```diff
@@ -58,3 +58,18 @@ def export_as_csv(modeladmin, request, queryset):  # pylint: disable=unused-argu
 
     export_as_csv.short_description = description
     return export_as_csv
+
+
+def get_clear_catalog_id_action(description=None):
```

Comment:
In fact I would suggest changing the file name altogether to something like `custom_django_admin_actions.py`.